### PR TITLE
Added empty function to jax.numpy frontend

### LINKED ISSUE #9218: empty

### PR DESCRIPTION
**Pull request by Mayank57**
Closes #9218

### CODE REVIEW

**Comment by Mayank57** [comment]
Hey, I am trying to link this PR to my issue but I am unable to find the settings icon beside the development option as shown in the Contributing video. Could you please help with this? I am trying to link this with #9218 issue

**Comment by Mayank57** [comment]
#9218

File: ivy/functional/frontends/jax/numpy/creation.py [hunk 1]
```diff
@@ -84,3 +84,9 @@ def eye(N, M=None, k=0, dtype=None):
 @to_ivy_arrays_and_back
 def triu(m, k=0):
     return ivy.triu(m, k=k)
+
+
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+def empty(shape, dtype=None):
+    return ivy.empty(shape, dtype=dtype)
```
Comment (kurshakuz) [inline ivy/functional/frontends/jax/numpy/creation.py:92]: Please add a new line character after this line. This currently causes lint error:
`W292 no newline at end of file`

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Hey! I have just linked your issue, so it should be fine now. I have also tested your implmentation locally and it passing tests. The only remaining thing is to fix the lint error as descirber below. Thanks!

**Review by kurshakuz — APPROVED** [review]
LGTM! Thank you for your contribution!

File: ivy/functional/frontends/jax/numpy/creation.py (hunk 1 above)
Comment (Mayank57) [reply ivy/functional/frontends/jax/numpy/creation.py]: Done. Added a new line character.

**Comment by Mayank57** [comment]
Thank you!

**Pull request merged**